planq/chain.json

```diff
@@ -33,19 +33,23 @@
   },
   "codebase": {
     "git_repo": "https://github.com/planq-network/planq",
-    "recommended_version": "v1.0.3",
+    "recommended_version": "v1.0.7",
```

Comment:
For planq, if v1.0.7 is incompatible with v1.0.0-v1.0.3 (consensus-breaking), then can we please add v1.0.7 as a new object under `versions` as well?

Reply:
yes please